packages/discord-snowflake/README.md

```diff
@@ -0,0 +1,104 @@
+# discord-snowflake
```

Comment:
Alternatively, @sapphire/snowflake :mm:

Reply:
I looked at it, and actually thought about using it for a bit! Ultimately wanted to use BigInt strings instead of classes though, less to rewrite